src/libsystemd-network/dhcp6-option.c

```diff
@@ -347,13 +347,13 @@ static int dhcp6_option_parse_pdprefix(DHCP6Option *option, DHCP6IA *ia,
         return 0;
 }
 
-int dhcp6_option_parse_ia(DHCP6Option *iaoption, DHCP6IA *ia) {
+int dhcp6_option_parse_ia(DHCP6Option *iaoption, DHCP6IA *ia, int *status_code) {
```

Comment:
The status is actually a 16bit unsigned, right? Hence I think the new param should be `uint16_t *ret_status_code`, no?